models/User.js

```diff
@@ -23,10 +23,11 @@ module.exports = function(sequelize, DataTypes) {
       type: DataTypes.STRING,
       allowNull: false,
     },
-    superuser: {
-      type: DataTypes.BOOLEAN,
-      defaultValue: false,
-    }
+    globalPermission: {
+      type: DataTypes.ENUM,
+      values: ['off', 'read', 'write'],
```

Comment:
Here's that list again :)

There should probably be constants in this file for the 3 possible values (which are just these string values, but this helps guard against typos) and then a const frozen array with the 3 possible values which can be used where the array is currently being hardcoded.

This helps to avoid errors and helps to find the locations where permissions are being referred to.